04_CollectionTypes.playground/Contents.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+/*
+ MARK: - 🏠 Домашнее задание.
+ 
+ TODO: - 🧑‍💻 Выполните каждый из пунктов для закрепления материала. Решение пишите на следующей строке после задания:
+ // - проинициализируйте два пустых массива foodArray and drinkArray разными способами
+ */
+//var foodArray: [String] = []
+//var drinkArray: [String] = []
+//взял в кавычки так как выдавало ошибку
+
+ // - присвойте первому массиву стринг значения - названия продуктов (минимум 5)
+var foodArray: [String] = ["burger", "sandwich", "frenchfries", "chocolate", "cheese", "sausage"]
+
+ // - присвойте второму массиву названия напитков (минимум 5)
+var drinkArray: [String] = ["milk", "cola" , "coffee", "beer" , "water", "juice"]
+
+ // - добавьте в конец foodArray еще одно значение
+foodArray.insert("butter", at: 6)
+
+ // - добавьте в середину drinkArray еще одно значение
+drinkArray.insert("coconut milk", at: 3)
+ 
+ // - создайте третий массив shoppingList, который должен включать в себя все значения из foodArray and drinkArray
+ var shoppingList = foodArray + drinkArray
+ 
+ // - распечатайте в консоль название продуктов и напитков из shoppingList и общее количество продуктов в shoppingList.
+print(shoppingList)
+print(shoppingList.count)
+ // - удалите первое и последнее значение, значение по индексу 2 из shoppingList
+shoppingList.remove(at:0)
+shoppingList.remove(at:12)
```

Comment:
what if shoppingList has more then 12 items or less ? 
your code must be independent of what u have, if array has 4 item ore 12... if condition is to add item in the middle u need to calculate this middle index